packages/website/src/widgets/header/header.test.tsx

```diff
@@ -0,0 +1,20 @@
+import '@testing-library/jest-dom'
+import { render, screen } from '@testing-library/react'
+import Header from './header'
+
+describe('Header', () => {
+  test('renders the header with branding, description, and action', async () => {
```

Comment:
This is a really good test. You followed RTL's best practices and wrote a clean test script. A solid start for the team.

nit. It would be better to split this test into multiple tests with each testing for one element.